### ISSUE TITLE
"Edit Card #n" removes one per line format

### ISSUE THREAD

**Issue by isthisnagee** [arkon/crcmaker#28]
Using Opera 38.0.2, Safari 9.1.1. When clicking Edit Card, it removes the one per line format. 
EX:

```
Responsibilities:
Resp1
Resp2
Resp3
```
becomes 
```
Responsibilities
Resp1,Resp2,Resp3
```
when edit is clicked.

**Comment by arkon** [comment]
Confirmed to be an issue in all browsers. I'm pretty sure I know where this is happening, so I'll try to get a fix in tonight.

**Comment by arkon** [comment]
Fixed in 9259e1fdbd866e0b6c2e01175d92c3167794a949
